test/kubemark/skeleton/util.sh

```diff
@@ -74,3 +74,20 @@ function copy-files() {
 function delete-master-instance-and-resources {
 	echo "Deleting master instance and its allocated resources" 1>&2
 }
+
+# Common colors used throughout the kubemark scripts
```

Comment:
This mimics the color coding in [hack/lib/util.sh](https://github.com/kubernetes/kubernetes/blob/master/hack/lib/util.sh#L778-L787) and is sourced by both [test/kubemark/start-kubemark.sh](https://github.com/kubernetes/kubernetes/blob/master/test/kubemark/start-kubemark.sh#L26) and [test/kubemark/stop-kubemark.sh](https://github.com/kubernetes/kubernetes/blob/master/test/kubemark/stop-kubemark.sh#L21).